docs/proposals/hpa-v2.md

```diff
@@ -0,0 +1,242 @@
+Horizontal Pod Autoscaler with Arbitary Metrics
```

Comment:
I can.  The reason I went with arbitrary metrics is because with most monitoring systems, it will also be possible to access built-in metrics, so, for instance, if a user wanted to scale on one of those directly, not as a resource percentage, this could be used).